app/helper.js

```diff
@@ -238,12 +237,17 @@ class YoutubeGrabberHelper {
 
           lengthSeconds = (minutes * 60) + seconds
         } else if (durationSplit[0] === 'SHORTS') { // durationText will still be 'SHORTS' for shorts
-          const regexMatch = video.title.accessibility.accessibilityData.label.match(shortsRegex)
-          lengthSeconds = parseInt(regexMatch[2])
-          durationText = '0:' + (lengthSeconds.toString().padStart(2, '0'))
-          if (regexMatch[3] === 'minute') {
+          const accessibilityData = video.title.accessibility.accessibilityData.label
+          const numbersAndSpacesRegex = /[^0-9\s]/g
+          const numbersOnly = accessibilityData.replace(numbersAndSpacesRegex, '').trim().split(' ').filter(number => {
+            return number !== ''
+          })
+          lengthSeconds = parseInt(numbersOnly[numbersOnly.length - 2])
```

Comment:
I can't understand this logic until I see an example value :S
```suggestion
          // Example value for 1 minute: `xxxxxx`
          // Example value for less than 1 minute: `xxxxxx`
          lengthSeconds = parseInt(numbersOnly[numbersOnly.length - 2])
```